apps/web/src/services/cost-explorer/store/cost-analysis-l-n-b-store.ts

```diff
@@ -0,0 +1,50 @@
+import { defineStore } from 'pinia';
+
+import { SpaceConnector } from '@cloudforet/core-lib/space-connector';
+
+import { store } from '@/store';
+
+import ErrorHandler from '@/common/composables/error/errorHandler';
+
+import type { CostQuerySetModel } from '@/services/cost-explorer/type';
+
+interface CostAnalysisLNBState {
+    costQueryList: CostQuerySetModel[];
+    dataSourceList: string[];
+    selectedQueryId?: string;
+    selectedDataSource?: string;
+}
+
+export const useCostAnalysisLNBStore = defineStore('cost-analysis-l-n-b', {
+    state: (): CostAnalysisLNBState => ({
+        costQueryList: [],
+        dataSourceList: [
+            'aws',
+            'azure',
+            'gcp',
+        ],
```

Comment:
nitpick: Please add TODO comment not to miss updating it and to notify reviewers that this is not the mistake. :)